app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd_list = command.split()
+    if cmd_list[1] != cmd_list[2]:
+        with (open(cmd_list[1], "r") as old_f,
+              open(cmd_list[2], "w") as new_f):
+            new_f.write(old_f.read())
```

Comment:
* Make sure you check that the `cp` input command is correct.
* Make sure that the length of `cmd_list` is three

Reply:
Done.